### ISSUE TITLE
Perlu Di Perjelas Keterangan Data Kependudukan Wilayah Pada Halaman Website Desa

### ISSUE THREAD

**Issue by muhtar-coder** [OpenSID/OpenSID#3844]
### Bagaimana alurnya sampai muncul masalah?
Tampilan Data kependudukan wilayah pada halaman website desa tidak bisa di pahami orang lain,karena tidak di cantumkan no urut RT dan tidak di ketahui siapa nama ketua RT nya.

### Seperti apa yang diharapkan?
Keterangan dapat mudah di pahami oleh orang lain saat membaca keterangan data kependudukan kewilayahan di halaman website desa menu statistik kependudukan sesuai kewilayahan.

### Apa yang terjadi?
- Data yang di tampilkan mudah di pahami orang ketika membaca di halaman website desa tanpa bertanya ke aparat desa,
- biasa nya para pelajar/mahasiswa/instansi lain sering meminta data keterangan jumlah penduduk desa per wilayah 
   RT/DUSUN/DESA.
### Informasi tambahan 
![image](https://user-images.githubusercontent.com/55665526/102799351-fe1f4a00-43e4-11eb-9bce-d124b1c926b9.png)

| Tanya            | Jawab
| ---------------- | ---
| Versi OpenSID    | 20.12
| Versi PHP        | 
| System operasi   |

**Comment by ilamazuliaf** [comment]
Punya kami versi premium juga membingungkan. https://kambinganbarat.com/first/wilayah

**Comment by mizwaruddin** [comment]
@afa28 Saya coba di demo OpenSID reguler dan premium sama-sama terjadi kekosongan data pada Nama Kepala Dusun. Saya juga sdh menguji dg mengganti dg tema klasik.

**Comment by afa28** [comment]
> @afa28 Saya coba di demo OpenSID reguler dan premium sama-sama terjadi kekosongan data pada Nama Kepala Dusun. Saya juga sdh menguji dg mengganti dg tema klasik.

Ok

**Comment by ilamazuliaf** [comment]
Saya coba edit themanya. Hasilnya seperti ini https://kambinganbarat.com/first/wilayah

**Comment by ariandii** [comment]
Apakah begini?

https://www.natairaya.desa.id/first/wilayah

<img width="960" alt="2021-01-07_14h38_04" src="https://user-images.githubusercontent.com/46939846/103865125-471a1280-50f6-11eb-912e-d8029fb05c86.png">

**Comment by muhtar-coder** [comment]
Ya... Dan mungkin pemisahan per dusun untuk lebih mudah di pahami.. Karena
penjumlahan ke atas sering orang gagal paham.

Pada tanggal Kam, 7 Jan 2021 14.40, Ariandi Ryan Kahfi <
notifications@github.com> menulis:

**Comment by afa28** [comment]
Apa dibuat collapse per wilayah agar tdk membingunkan???

**Comment by andifahruddinakas** [comment]
pak @eddieridwan ini perlu di buat kesimpulan pembahasan.

**Comment by eddieridwan** [comment]
@afa28 , seperti berikut mungkin lebih jelas.

![wilayah-1](https://user-images.githubusercontent.com/249563/136011962-bef5f617-1970-4072-891f-04930b977022.jpg)

**Comment by muhtar-coder** [comment]
Setuju...mudah di pahami...saat ini yg paling sering minta data penduduk
per rt adalah kader stunting,kader posyandu ..dll

Pada tanggal Sel, 5 Okt 2021 18.10, Eddie Ridwan ***@***.***>
menulis:

**Comment by andifahruddinakas** [comment]
> @afa28 , seperti berikut mungkin lebih jelas.
> 
> ![wilayah-1](https://user-images.githubusercontent.com/249563/136011962-bef5f617-1970-4072-891f-04930b977022.jpg)

ok.

ini dihalaman web saja atau di halaman admin jg???

**Comment by sukrimalendra** [comment]
Jika boleh usul, tampilkan juga jumlah KK laki-laki dan perempuan. Karena sekarang banyak yg minta Jumlah KK Laki-laki dan perempuan.

**Comment by agungsugiarto** [comment]
Sudah diperbaiki melalui pr https://github.com/OpenSID/premium/pull/434
